### PR TITLE
Add _trackTimings

### PR DESCRIPTION
**Pull request by IdoHakimi**

### CODE REVIEW

**Comment by revolunet** [comment]
Thanks!
Can you please add an example in the README ?

**Comment by IdoHakimi** [comment]
sure thing

**Comment by justinsa** [comment]
Even though this is a trivial function, having a single test for it that confirms calling trackTimings results in _send being called with the appropriate values and order would mitigate future changes accidentally modifying the behavior.

**Comment by IdoHakimi** [comment]
just added trackTimings test, please merge to master :)

**Comment by revolunet** [comment]
thanks you !

File: test/unit/angular-google-analytics.js [hunk 1]
```diff
@@ -581,7 +581,7 @@ describe('angular-google-analytics', function () {
   describe('supports arbitrary page events', function () {
     beforeEach(module(function (AnalyticsProvider) {
       AnalyticsProvider.setPageEvent('$stateChangeSuccess');
-    }));
+    }));e
```
Comment (revolunet) [inline test/unit/angular-google-analytics.js]: ?

**Pull request merged**